package.json

```diff
@@ -18,7 +18,7 @@
   "dependencies": {
     "@reduxjs/toolkit": "^1.6.2",
     "@types/react": "^17.0.15",
-    "@yext/answers-headless": "^0.1.0-beta.4",
+    "@yext/answers-headless": "file:~/answers-headless",
```

Comment:
Just confirming we're gong to publish a new headless version and update this before merging

Reply:
Yup!